### ISSUE TITLE
WCAG 4.1.2: Buttons must have discernible text (#decrement) (#increment)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#117]
The irony of thee accessibility code failing basic accessibility ;)

![image](https://user-images.githubusercontent.com/1296369/69916160-5c326580-144f-11ea-8594-c1cf704a13b1.png)

#### Issue
`Buttons must have discernible text` ([button-name](https://dequeuniversity.com/rules/axe/3.3/button-name?application=msftAI))
#### Target application
[Control Panel - 109 - Administration](http://localhost/kh/administrator/index.php)
#### Element path
#decrement
#### Snippet
`<button class="font-unresizable icon-minus-sign" id="decrement"></button>`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"
      Element has no title attribute or the title attribute is empty
